docs/ert/reference/configuration/syntax.rst

```diff
@@ -0,0 +1,22 @@
+The ert configuration file format
+=================================
+
+The ert config file, workflows, workflow job configuration files and forward
+model configuration files all share the same syntax. These are lines starting
+with one keyword followed by arguments separated by whitespace. e.g.
+
+::
+
+        QUEUE_SYSTEM LOCAL
+        NUM_REALIZATIONS 12
+        -- A comment
```

Comment:
Maybe add to the text that -- is used for comments as well?

Reply:
Added.